src/repository/RepositoryUsers.js

```diff
@@ -0,0 +1,25 @@
+const { User } = require('../models');
+
+const findByEmail = async (email) => {
+  const findEmail = await User.findOne({ where: { email } });
+
+  return findEmail;
+};
+
+const findByNickName = async (nickname) => {
```

Comment:
```suggestion
const findByNickname = async (nickname) => {
```
nickname é uma palavra só

Reply:
corrigido nome da variavel